### ISSUE TITLE
Manual Install for use with Steam Version on Linux

### ISSUE THREAD

**Issue by Severus157** [sleepeybunney/maelstrom#42]
Hi,
Did someone has had some Luck with a manual installation or some other information helpful to use for getting the Mod running on Linux, so it is usable with the Steam Version on Proton?

**Comment by CocaineJohnsson** [comment]
Yes, it works. It doesn't work under proton though (proton makes 64-bit prefixes by default and while you can re-make the prefix as 32-bit, this is too much of a headache) but it does work under wine 8.1.

Firstly you **NEED** a 32-bit prefix, a 64 bit one will not work. If your default `~/.wine` prefix isn't 32-bit you can either delete it and recreate it with `WINEARCH="win32" winecfg` or you can specify an alternate prefix with `WINEPREFIX=/path/to/prefix WINEARCH="win32"`, either way works. I opted for the latter since my default prefix is 64-bit on purpose.

corefonts is REQUIRED.
dotnet >= 4.0 is REQUIRED (tested 4.7.2 and 4.8)
ddr=gdi is REQUIRED (it crashes immediately without it)

I found this to work fairly well:

Set up the wine prefix, this will take several minutes. Just wait for it to finish.
`WINEPREFIX=~/.local/share/wineprefixes/Maelstrom WINEARCH="win32" winetricks -q corefonts dotnet48 ddr=gdi win7 `

Run Maelstrom.
`WINEPREFIX=~/.local/share/wineprefixes/Maelstrom wine Maelstrom.exe`


If you want to be able to just double click the EXE you have to use the default prefix, `~/.wine`. Since it likely already exists, delete it and follow the instructions above, then use `winetricks -q corefonts dotnet48 ddr=gdi win7` to install the same packages as before.

**Comment by CocaineJohnsson** [comment]
Oh and to actually configure MAELSTROM you might want to run winecfg in this prefix and select "show dot files" since steam by default installs games to a hidden directory.

![image](https://user-images.githubusercontent.com/53689060/221377231-8d9924b3-872c-4f41-b8a4-7d7325f2390a.png)
